our_site/planner/templates/planner/event_home.html

```diff
@@ -29,6 +31,36 @@ <h1 class="title is-1">{{ event.name }}</h1>
         </div>
     </div>
 
+    <div class="container">
+        <div class="columns">
+            <div class="column is-2">
+                <h2>Invitees:</h2>
+                <ul>
+                    {% for invitee in invitees %}
+                      <li>{{ invitee.user.username }}</li>
+                    {% endfor %}
+                </ul>
+            </div>
+            <div class="column is-6">
+                <h2>Attendees:</h2>
+                <ul>
+                    {% for attendee in attendees %}
+                        <li>{{ attendee.user.username }}</li>
+                    {% endfor %}
+                </ul>
+            </div>
+        </div>
+    </div>
+
+
+
+    {% if user.profile in invitees %}
+        <form method="post" action="{% url 'planner:handle_event' event.id %}">
+            {% csrf_token %}
+            <button type="submit" name="action" value="accept">Accept Invite</button>

```

Comment:
not needed but would be nice if the buttons had the same css as all the others just so it looks uniform